styles.css

```diff
@@ -9,6 +9,11 @@ body {
   background-color: #f1f1f1;
 }
 
+header {
+  position: sticky;
+  top: 0;
+}
+
 .imageone {
```

Comment:
[OPTIONAL] Great job with the close menu button. But unfortunately, previous reviewer didn't notice the color of nav hamburger icon. Kindly fix the color of it to make your design more coherent 👍  
<img width="243" alt="Screenshot 2023-02-06 at 10 41 16 PM" src="https://user-images.githubusercontent.com/25887253/217069537-a7ff900d-6001-4a96-b75b-3719f6430919.png">

<img width="356" alt="Screenshot 2023-02-06 at 10 41 42 PM" src="https://user-images.githubusercontent.com/25887253/217069098-5cfa269a-62fc-47bc-9b8f-1b1ae0ae4e50.png">